src/chunk_append/planner.c

```diff
@@ -17,15 +17,20 @@
 #include <optimizer/prep.h>
 #include <optimizer/subselect.h>
 #include <optimizer/tlist.h>
-#include <optimizer/var.h>
 #include <parser/parsetree.h>
 
+#include "compat.h"
+#if PG12_LT
+#include <optimizer/var.h> /* f09346a */
```

Comment:
What does the hash number mean here (commit ID)? Please clarify.

Reply:
It's commit hash for Postgre GitHub repo and goes to https://github.com/postgres/postgres/commit/f09346a9c6218dd239fdf3a79a729716c0d305bd
I can check how it will look like in GH if the link is used.

Reply:
Or may be to write just `postgres/postgres@f09346a`in the code instead of `https://github.com/postgres/postgres/commit/f09346a9c6218dd239fdf3a79a729716c0d305bd`